sequence_task_2/student/include/impl.h

```diff
@@ -14,4 +15,7 @@
  * @note See https://en.cppreference.com/w/cpp/algorithm/rotate
  */
 template<class T, class BI>
-void rotate(std::list<T>& list, BI begin, BI new_begin, BI end);
+void rotate(std::list<T>& list, BI begin, BI new_begin, BI end)
+{
+    std::rotate(begin, new_begin, end);
```

Comment:
Please read task description

Reply:
list.splice(new_begin, list, begin, end);
is not work for me!!!
test are failed
maintest.cpp:12: Failure
Expected equality of these values:
  data
    Which is: {}
  expected
    Which is: { 1, 2, 3, 4, 5, 0 }

rotated with empty list